src/seam-connect/client.ts

```diff
@@ -57,7 +59,10 @@ export const getSeamClientOptionsWithDefaults = (
       globalThis?.process?.env?.SEAM_WORKSPACE_ID ??
       undefined,
     axiosOptions: providedOptions.axiosOptions ?? {},
-    clientSessionToken: providedOptions.clientSessionToken ?? undefined,
+    clientSessionToken:
+      providedOptions.clientSessionToken === null
+        ? null
+        : providedOptions.clientSessionToken ?? null,
```

Comment:
This or the other suggestion so it's consistent. Probably this one.
```suggestion
        : providedOptions.clientSessionToken ?? undefined,
```